doc/whats_new/upcoming_changes/sklearn.decomposition/30443.enhancement.rst

```diff
@@ -0,0 +1,2 @@
+- :class:`~sklearn.decomposition.DictionaryLearning` and :class:`~sklearn.decomposition.SparseCoder` now have a
+  ``inverse_transform`` method. By :user:`Rémi Flamary <rflamary>`
```

Comment:
`MiniBatchDictionaryLearning` should be listed as well since it inherits from `_BaseSparseCoding`.

Also we tend to classify this as a feature rather than an enhancement.